build.gradle

```diff
@@ -103,6 +103,7 @@ subprojects {
         toolVersion = detektPluginVersion
 
         config = files('../detekt.yml')
+        input = files('src/main/kotlin', 'src/test/kotlin', 'src/funTest/kotlin')
```

Comment:
Even if we don't have it shall we keep the  '"src/main/java' ? Asking because the commit message sounds a bit like removing '"src/main/java"' might not have been intended.

Reply:
I don't think we should keep it, and yes, removal was intended 😉